.github/workflows/regression-tests.yml

```
@@ -25,7 +25,7 @@ jobs:
           - {pgver: 11, devpkg: 11}
           - {pgver: 12, devpkg: 12}
           - {pgver: 13, devpkg: 13}
-          - {pgver: 14beta3, devpkg: 14}
+          - {pgver: 14, devpkg: 14}
```

Since these are all now, `pgver` == `devpkg`, can we do something like this and drop the verbose entries (L21-L28)?

```suggestion
          pgver: [9.4, 9.5, 9.6, 10, 11, 12, 13, 14]
```


and then in the Dockerfile: 
https://github.com/pgaudit/set_user/blob/master/.github/docker/Dockerfile#L6

`ARG DEVPKG=PGVER`

---

Addressed in https://github.com/pgaudit/set_user/pull/54/commits/02d865a6851a9e95dbedad4afeed6c0a9eaa0d75